lib/routes/avatar/upload.js

```diff
@@ -7,16 +7,20 @@ const assert = require('assert');
 const Joi = require('joi');
 
 const config = require('../../config');
+const logger = require('../../logging')('routes.avatar.upload');
 const db = require('../../db');
 const hex = require('buf').to.hex;
 const img = require('../../img');
+const SQSSender = require('../../sqs')(logger);
 const validate = require('../../validate');
 const workers = require('../../img-workers');
 
 const FXA_PROVIDER = 'fxa';
 const FXA_URL_TEMPLATE = config.get('img.url');
 assert(FXA_URL_TEMPLATE.indexOf('{id}') !== -1,
     'img.url must contain the string "{id}"');
+const profileUpdatedQueue = new SQSSender(config.get('authServerMessaging.region'),
+                                          config.get('authServerMessaging.profileUpdatesQueueUrl'));
```

Comment:
I wonder if we could push this down a level of abstraction, so we can create this `SQSSender` object in one place, and then all these other modules just have to import something and pass it the target uid.